templates/kubernetes/index.html

```diff
@@ -1,665 +1,818 @@
-            hi_def=True,
-            loading="lazy",
-            attrs={"class": "p-logo-section__logo"}
-            ) | safe
-          }}
+      </div>
+      <div class="col">
+        <div class="p-section--shallow u-hide--medium u-hide--small">
+          <h2>
+            Public cloud Kubernetes
+            <br />
+            powered by Ubuntu
+          </h2>
         </div>
-        <div class="p-logo-section__item">
-          {{ 
-            image (
-            url="https://assets.ubuntu.com/v1/05285685-equinix-logo.png",
-            alt="Equinix",
-            width="113",
-            height="135",
-            hi_def=True,
-            loading="lazy",
-            attrs={"class": "p-logo-section__logo"}
-            ) | safe
-          }}
+        <p>Cloud-tailored experience, optimized kernels, unlimited options</p>
+        <hr class="p-rule--muted u-no-margin--bottom" />
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Ubuntu is the host OS for the world's most popular cloud-hosted Kubernetes</li>
+          <li class="p-list__item is-ticked">Optimized for performance, boot speed, and drivers on all major public clouds</li>
+          <li class="p-list__item is-ticked">
+            Out-of-the-box cloud integration with the option of enterprise-grade commercial support
+          </li>
+          <li class="p-list__item is-ticked">Ubuntu Pro provides deeper integrations with cloud services</li>
+          <li class="p-list__item is-ticked">In-cloud Ubuntu package mirrors for high bandwidth and local updates</li>
+          <li class="p-list__item is-ticked">Full control over kernel security patching with Canonical Livepatch</li>
+        </ul>
+        <div class="p-cta-block">
+          <p>
+            <a href="/public-cloud">Learn more about Ubuntu on public clouds&nbsp;&rsaquo;</a>
+          </p>
         </div>
-        <div class="p-logo-section__item">
-          {{ 
-            image (
-            url="https://assets.ubuntu.com/v1/dae49585-joyent-logo.png",
-            alt="Joyent",
-            width="105",
-            height="135",
-            hi_def=True,
-            loading="lazy",
-            attrs={"class": "p-logo-section__logo"}
-            ) | safe
-          }}
+        <div class="p-logo-section--dense">
+          <div class="p-logo-section__items">
+            <div class="p-logo-section__item">
+              {{ image(url="https://assets.ubuntu.com/v1/15f4f759-aws-logo.png",
+                            alt="AWS",
+                            width="151",
+                            height="312",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-logo-section__logo"}) | safe
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image(url="https://assets.ubuntu.com/v1/f1ec4ff5-microsoft-azure-logo%20[DEPRECATED]%20.png",
```

Comment:
The link to this image would suggest we shouldn't use it?

Reply:
I raised this with @mattea-turic and she said we are OK to use them